core/src/main/java/org/apache/accumulo/fate/Fate.java

```diff
@@ -70,6 +65,29 @@
 
   private final AtomicBoolean keepRunning = new AtomicBoolean(true);
 
+  public enum NodeData {
+
+    TARGET_NODE("target_node"),
```

Comment:
I think we can remove `_node` from all of the enum names. Also, I would support renaming `Repo.getDescription` to `Repo.getName` or `Repo.getClassName` as the class name is only ever used. Then, instead of using `target` here, I would just use `name`.

Reply:
Without `_node` the exception and return nodes seem like the should have some decoration.  The `target` (formally prop_debug) is the classname of the REPO that is the "head" of the transaction (the "tail" of the stack) - the REPO that starts things, but can have sub-steps.   The distinction of that "name" should be reflected in the code.  Things like repoHeadName seem awkward.  To contrast with top that is the currently executing transaction, tail - but then that requires a deep understanding of the stack nature of REPO.

Reply:
If you take my comment about `Fate.NodeData` into account and rename the enum, then you would end up with something like:
```
enum TxInfo {
  NAME, EXCEPTION, RETURN, AUTO_CLEAN;
}
```

> Without _node the exception and return nodes seem like the should have some decoration.

I think `TxInfo.EXCEPTION` is pretty clear. Maybe `TxInfo.RETURN_VALUE` is a little more clear.

Reply:
Both `TxInfo.EXCEPTION` and `TxInfo.RETURN_VALUE` seem good.  `NAME` feels to generic - what about `TxInfo.REPO_NAME`

Reply:
> what about TxInfo.REPO_NAME

Sure, IMO better than `target` and `debug`. I would still suggest renaming `Repo.getDescription`, maybe in this case to `Repo.getName`.

Reply:
@cshannon - @EdColeman and I talked offline and we settled on the following structure for the enum:
```
TxInfo {
  REPO_TARGET, EXCEPTION, RETURN_VALUE, AUTO_CLEAN
}
```

Reply:
Did you guys mean for the first value in the Enum to be "REPO_NAME" instead? While working on the new Fate API, I came to parts of the code using the field and I found that the term "target" doesn't make sense. https://github.com/apache/accumulo/blob/e08287065d3af4773dd97a3991686abc3002f6dd/core/src/main/java/org/apache/accumulo/fate/Fate.java#L303

Reply:
The description [here](https://github.com/apache/accumulo/blob/e08287065d3af4773dd97a3991686abc3002f6dd/core/src/main/java/org/apache/accumulo/fate/AdminUtil.java#L108) made me scratch my head. When we print out the value on the CLI, its just referred to as "op", meaning the name of the current operation. https://github.com/apache/accumulo/blob/e08287065d3af4773dd97a3991686abc3002f6dd/core/src/main/java/org/apache/accumulo/fate/AdminUtil.java#L421-L425

Here is a Fate Tx printed for a compaction:
<pre>
txid: 2d4b1f3195963320  status: IN_PROGRESS         op: CompactRange     locked: [R:+default, R:1] locking: []              top: CompactionDriver created: 2022-08-30T19:30:00.06Z
</pre>

So "CompactRange" is really just the name of the current operation running for the Tx.

Reply:
In the comment thread above, `name` and `repoName` were both suggested. I don't remember how we ended up at repoTarget.

Reply:
I made a fix in #2904